tests/components/sensor/test_srp_energy.py

```diff
@@ -0,0 +1,73 @@
+"""The tests for the Srp Energy Platform."""
+import unittest
+from unittest.mock import patch
+from homeassistant.setup import setup_component
+
+from tests.common import (get_test_home_assistant,
+                          MockDependency)
+
+VALID_CONFIG_MINIMAL = {
+    'sensor': {
+        'platform': 'srp_energy',
+        'username': 'foo',
+        'password': 'bar',
+        'id': 1234
+    }
+}
+
+
+def mock_init(self, accountid, username, password):
+    """Mock srpusage usage."""
+    return None
+
+
+def mock_usage(self, startdate, enddate):  # pylint: disable=invalid-name
+    """Mock srpusage usage."""
+    usage = [
+        ('9/19/2018', '12:00 AM', '2018-09-19T00:00:00-7:00', '1.2', '0.17'),
+        ('9/19/2018', '1:00 AM', '2018-09-19T01:00:00-7:00', '2.1', '0.30'),
+        ('9/19/2018', '2:00 AM', '2018-09-19T02:00:00-7:00', '1.5', '0.23'),
+        ('9/19/2018', '9:00 PM', '2018-09-19T21:00:00-7:00', '1.2', '0.19'),
+        ('9/19/2018', '10:00 PM', '2018-09-19T22:00:00-7:00', '1.1', '0.18'),
+        ('9/19/2018', '11:00 PM', '2018-09-19T23:00:00-7:00', '0.4', '0.09')
+        ]
+    return usage
+
+
+class TestSrpEnergySetup(unittest.TestCase):
```

Comment:
We are migrating all tests to pytest based standalone test functions. Please adhere to that for these tests.

Reply:
What do you mean standalone test function? What changes do I need to make?

Reply:
Here's a simple example:
https://github.com/home-assistant/home-assistant/blob/dev/tests/components/sensor/test_deconz.py#L106